integration-tests/src/test/java/org/web3j/generated/Arrays.java

```diff
@@ -20,7 +20,7 @@
  * or the org.web3j.codegen.SolidityFunctionWrapperGenerator in the 
  * <a href="https://github.com/web3j/web3j/tree/master/codegen">codegen module</a> to update.
  *
- * <p>Generated with web3j version 3.0.0-alpha4.
+ * <p>Generated with web3j version 3.0.0.
```

Comment:
Seemed like the alpha4 was not needed anymore, but it could have been a mistake - let me know.

Reply:
It should be picked up by the merge from the master branch.

Reply:
I guess I got carried away here, as I did not completely understand the process between the master and the android branches... feel free to close this branch if this is going to cause problems later on.

Reply:
It may be easier just to start fresh - my usual workflow is simply:

```
git checkout web3j-android
git fetch origin
git merge origin/master
```
Then fix any merge errors.

I try to ensure any non-compatibility changes are made to the master branch first.

Reply:
I'll merge your other PR fixing up the warnings - once done, do you fancy updating the Android branch with the suggestion you made for SafeVarArgs?

Reply:
Sounds good!  I don't really have a way to test on Android either, but I think this should work.  If not, it's easy to revert just the SafeVarargs part.

Reply:
The commit I've just done is for the 3.1.0 release - please merge onto Android and let us know once it's done so I can get the Android release out.